docs/getting-started/brokers/sqs.rst

```diff
@@ -14,7 +14,7 @@ library using :command:`pip`:
 
 .. code-block:: console
 
-    $ pip install -U boto3
+    $ pip install celery[sqs]
```

Comment:
Could you please update the preceding note about `boto` too? Thanks!

Reply:
Done, I've used the one for Redis as a template, because it also uses bundle.